src/main/java/uk/gov/pay/api/model/PaymentError.java

```diff
@@ -15,7 +15,9 @@
         P0100,
         P0101,
         P0102,
-        P0103
+        P0103,
+        P0401,
+        P0498
```

Comment:
This is a general comment about the PaymentError class. Do you think it makes sense to model this in a way where we capture a description of the Code within the Code enum. Looking at the Code enum the code doesn't give out any description unless you look at the different static methods. But that too the description is an argument.